scripts/requirements.txt

```diff
@@ -1,5 +1,4 @@
 ecdsa
-editdistance>=0.5.0
 imagesize>=1.2.0
 intelhex
 pygit2>=0.26.0
```

Comment:
I think this should be removed, right?

Reply:
Yes, sorry, I forgot `git add -u` :|
Thanks!